docs/_static/Makefile

```diff
@@ -34,7 +34,7 @@ pedigree3.svg : pedigrees.ink.svg
 pedigree_recapitate.svg : pedigrees.ink.svg
 	./export-layers-svg.sh $< \
 		ts_indivs simp_indivs pedigree simp_pedigree chromosomes simplify_removes \
-		recap_pedigree extra_chromosomes tskit_time recap_label recap \
+		recap_pedigree tskit_time recap_label recap \
```

Comment:
Did you mean to do this? Here's the change:
![Screenshot from 2021-01-18 08-37-21](https://user-images.githubusercontent.com/1046249/104941887-85ba9300-5968-11eb-8163-c41bbf3630c3.png)
![Screenshot from 2021-01-18 08-37-28](https://user-images.githubusercontent.com/1046249/104941891-86ebc000-5968-11eb-8f87-66f2d31fa796.png)

Reply:
I did mean to do this because I thought you would want the picture in the recapitated sequence to match the ones in part B of pedigree01.png? It seemed very odd that it included the extra (simplified-away) chromosomes. Perhaps I misunderstood something?

Reply:
Ah, I see. Good call.